backend/src/api/routes/unified_guide.py

```
@@ -0,0 +1,154 @@
+        
+        return {
+            "success": True,
+            "guide": guide,
+            "validation": {
+                "is_valid": is_valid,
+                "details": details
+            },
+            "generated_with": "unified_guide_service"
+        }
+        
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)